stixcore/idb/manager.py

```diff
@@ -237,27 +237,37 @@ def convert_mib_2_sqlite(*, in_folder, out_file, version_label):
                         nextID = nr
                 nextID += 1
 
-                # NIX00405
-                nix = 'NIX00405'
-                count,  = cur.execute("select count(*) from PCF where PCF_NAME = ? " +
-                                      "AND PCF_CURTX not NULL", (nix,))\
-                             .fetchone()
-                if count == 0:
-                    pname, nextID = IDB.generate_calibration_name("CIX", nextID)
-                    cur.execute('''update PCF set
-                                    PCF_CURTX = ?,
-                                    PCF_CATEG = 'N',
-                                    PCF_UNIT = 's'
-                                   where PCF_NAME = ?''', (pname, nix))
-
-                    cur.execute('''insert into MCF (MCF_IDENT, MCF_DESCR, MCF_POL1,
-                                    MCF_POL2, MCF_POL3, MCF_POL4, MCF_POL5, SDB_IMPORTED)
-                                   values
-                                    (?,?,?,?,?,?,?, 0)''', (pname, "Duration", 0, 0.1, 0, 0, 0))
-
-                else:
-                    logger.info(f"Skip calib injection for {nix}: allready present")
-                print(nix)
+                # inject polynomial calibrations
+
+                duration = ("duration", 0, 0.1, 0, 0, 0)
+                duration_p1 = ("duration + 0.1", 0.1, 0.1, 0, 0, 0)
+                binary_seconds = ("binary seconds", 0, 1.0 / 65536, 0, 0, 0)
```

Comment:
I think this should be 1.0/(2^16-1) = 1.0/65535

Reply:
fixed